From d32e64abf2443ada592f6a1e5d74bee35db0f221 Mon Sep 17 00:00:00 2001
From: "Laszlo Boszormenyi (GCS)" <gcs@debian.org>
Date: Sat, 9 Sep 2017 07:57:16 +0000
Subject: [PATCH] Don't clobber 'r2' register on ppc64el

It was invalid even before, now GCC 7 no longer accept this.
---
 platform/switch_ppc64_linux.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platform/switch_ppc64_linux.h b/platform/switch_ppc64_linux.h
index 0f20756..cd7d748 100644
--- a/platform/switch_ppc64_linux.h
+++ b/platform/switch_ppc64_linux.h
@@ -56,7 +56,7 @@
 #define ALTIVEC_REGS
 #endif
 
-#define REGS_TO_SAVE "r2", "r14", "r15", "r16", "r17", "r18", "r19", "r20", \
+#define REGS_TO_SAVE "r14", "r15", "r16", "r17", "r18", "r19", "r20", \
        "r21", "r22", "r23", "r24", "r25", "r26", "r27", "r28", "r29", "r31", \
        "fr14", "fr15", "fr16", "fr17", "fr18", "fr19", "fr20", "fr21", \
        "fr22", "fr23", "fr24", "fr25", "fr26", "fr27", "fr28", "fr29", \
-- 
2.15.1

